src/plone/restapi/serializer/discussion.py

```diff
@@ -1,12 +1,18 @@
 # -*- coding: utf-8 -*-
-from plone.app.discussion.interfaces import IConversation
 from plone.app.discussion.interfaces import IComment
+from plone.app.discussion.interfaces import IConversation
 from plone.restapi.batching import HypermediaBatch
-from zope.publisher.interfaces import IRequest
-from plone.restapi.interfaces import ISerializeToJson
 from plone.restapi.interfaces import IJsonCompatible
-from zope.component import adapter, getMultiAdapter
+from plone.restapi.interfaces import ISerializeToJson
+from plone.restapi.services.discussion.utils import can_delete
```

Comment:
@sneridagh I guess this is a matter of taste but I usually prefer to keep the stuff in one file. No other class is using the methods in utils, so we do not need to factor them out. Therefore the dev has to open two files instead of just one.